chapters/VK_EXT_debug_utils.txt

```diff
@@ -331,55 +331,53 @@ include::../validity/protos/vkCmdInsertDebugUtilsLabelEXT.txt[]
 [[debugging-debug-messengers]]
 === Debug Messengers
 
-Vulkan allows an application to register multiple callbacks with any Vulkan
-component wishing to report debug information.
-Some callbacks may log the information to a file, others may cause a debug
-break point or other application defined behavior.
-A primary producer of callback messages are the validation layers.
-An application can: register callbacks even when no validation layers are
-enabled, but they will only be called for the Vulkan loader and, if
-implemented, other layer and driver events.
+Vulkan allows an application to register arbitrary number of callbacks with
+all the Vulkan components wishing to report debug information.
+Some callbacks can: log the information to a file, others can: cause a debug
+break point or any other behavior defined by the application.
+A primary producer of callback messages are the
+link:++https://github.com/KhronosGroup/Vulkan-ValidationLayers++[validation
```

Comment:
Isn't that Markdown?? Well, I will consult a styleguide when I get a chance...

BTW I wasn't sure if the author meant **a** validation layers, or **the** Validation Layer. But probably doesn't matter; this paragraph feels informational/introductory...

Reply:
Good point, I'm not sure if it is.  That's the way I've been doing it, and see it done elsewhere in our docs.

We have more than one validation layer so I think it's valid to keep the "s".  They're just grouped together under "VK_LAYER_LUNARG_standard_validation", but there are multiple.  Plus, I think IHVs may create their own (or at least could).

Reply:
Sorry typo with the s. I mean if it meant validation layers in general, or The KhronosGroup Validation Layers. Potentially the former, as "validation" is even one of the msg Types.

Your link markup seems undocumented by asciidoctor. It would seem like that it was used that way in the spec, because the parens are actual text rendered, but looks like Markdown, so understandable mistake.